plaso/parsers/sqlite.py

```diff
@@ -273,6 +274,13 @@ def Open(self, file_object, wal_file_object=None):
           table_name: ' '.join(query.split())
           for table_name, query in sql_results}
 
+      for table in self.schema.keys():
```

Comment:
Preference nit:
table => table_name

since it is not a table object or equiv.